src/parser/parser.yy

```diff
@@ -1523,6 +1530,7 @@ maintain_sentence
     | revoke_sentence { $$ = $1; }
     | get_config_sentence { $$ = $1; }
     | set_config_sentence { $$ = $1; }
+    | balance_sentence { $$ = $1; }
```

Comment:
`mutate_sentence` is included the DML such as insert and delete.

Should we move `balance_sentence` into `maintain_sentence` ?

Reply:
good suggestion